Library/Homebrew/cmd/commands.rb

```diff
@@ -6,53 +6,63 @@
 
 module Homebrew
   module_function
-
   def commands
-    if ARGV.include? "--quiet"
-      cmds = internal_commands + external_commands
-      cmds += internal_developer_commands
-      cmds += HOMEBREW_INTERNAL_COMMAND_ALIASES.keys if ARGV.include? "--include-aliases"
-      puts Formatter.columns(cmds.sort)
-    else
-      # Find commands in Homebrew/cmd
-      puts "Built-in commands"
-      puts Formatter.columns(internal_commands)
-
-      # Find commands in Homebrew/dev-cmd
-      puts
-      puts "Built-in developer commands"
-      puts Formatter.columns(internal_developer_commands)
-
-      # Find commands in the path
-      unless (exts = external_commands).empty?
+    Homebrew::InstallCommand.commands
+  end
+
+  class InstallCommand < CommandOptions
```

Comment:
I think `CommandsCommand` better reflects what it does, since we'll probably also have an `InstallCommand` class eventually (based on what was said earlier about one class per command).